<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>503</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>student manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>system icon.png</normaloff>system icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>11</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.463687, y1:0.505, x2:0.818, y2:0.011, stop:0.397727 rgba(191, 220, 255, 66), stop:1 rgba(255, 255, 255, 255));
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget_remove"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_confirm_remove">
            <property name="text">
             <string>confirm remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableWidget" name="tableWidget_modify"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="action_new_student"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="action_Search_By_Name"/>
    <addaction name="action_Search_By_Gender"/>
    <addaction name="action_Search_By_Age"/>
    <addaction name="action_Search_By_ID"/>
   </widget>
   <widget class="QMenu" name="menuDelete">
    <property name="title">
     <string>Remove</string>
    </property>
    <addaction name="action_remove_list"/>
   </widget>
   <widget class="QMenu" name="menuModify">
    <property name="title">
     <string>Modify</string>
    </property>
    <addaction name="action_modify_list"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionAll_Students_Info"/>
   </widget>
   <addaction name="menuAdd"/>
   <addaction name="menuSearch"/>
   <addaction name="menuDelete"/>
   <addaction name="menuModify"/>
   <addaction name="menuShow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_new_student">
   <property name="text">
    <string>new student</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAll_Students_Info">
   <property name="text">
    <string>All Students Info</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Search_By_Name">
   <property name="text">
    <string>By Name</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Search_By_Gender">
   <property name="text">
    <string>By Gender</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Search_By_Age">
   <property name="text">
    <string>By Age</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Search_By_ID">
   <property name="text">
    <string>By ID</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_remove_list">
   <property name="text">
    <string>remove list</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_modify_list">
   <property name="text">
    <string>modify list</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
